pkgs/tools/misc/empty/default.nix

```
@@ -0,0 +1,50 @@
+{ fetchgit, lib, stdenv, which }:
+
+let
+  pname = "empty";
+  rev = "6e81f477dba6c2f9086da70b02abec71b64fb16e";
+  baseVersion = "0.6.21b";
+  version = baseVersion + "+1-git-" + (builtins.substring 0 7 rev);
+in stdenv.mkDerivation {
+  inherit pname version;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "empty";
  version = "unstable-XXXX-XX-XX";
```
where XXXX-XX-XX the date of the commit.

---

Packaged the latest release instead of the latest commit.